deploy/install.sh

```
@@ -59,65 +51,76 @@ function installing_onboarder() {
+      echo "s3_bucket not provided; EXITING;"
+      exit 1
     fi
+
     if [[ $s3_bucket == *[' !@#$%^&*()+']* ]]; then
-      echo "s3_bucket should not contain spaces / any special character; EXITING";
-      exit 1;
+      echo "s3_bucket should not contain spaces / any special character; EXITING"
+      exit 1
     fi
+
     read -p "Provide onboarder s3 bucket region : " s3_region
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)